packages/localization/src/DefaultVocabulary.js

```diff
@@ -52,6 +50,31 @@ const Translations = {
     'No internet connection, please check your internet settings or try it later.',
 };
 
+const Translations = {
+  ...HotelsVocabulary,
+  ...ManageMyBookingVocabulary,
+  ...SharedVocabulary,
+};
+
 export type TranslationKeys = $Keys<typeof Translations>;
 export type TranslationKeysObject = { [TranslationKeys]: string };
 export default (Translations: TranslationKeysObject);
+
+export type TranslationPackages =
+  | 'HotelsVocabulary'
+  | 'ManageMyBookingVocabulary'
+  | 'SharedVocabulary';
+
+export const getVocabularies = (vocabularies: TranslationPackages[]) => {
+  const translationPackages = {
+    HotelsVocabulary: { ...HotelsVocabulary },
```

Comment:
Why is this necessary? Why not just `HotelsVocabulary`? I am not sure I understand it.